api/database/migrations/2020_09_08_190017_make_event_type_id_nullable_custom_fields_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class MakeEventTypeIdNullableCustomFieldsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('custom_fields', function (Blueprint $table) {
+            $table->dropColumn('event_type_id');
```

Comment:
We should be able to update it without deleting

Reply:
How?
I did it because, if we edit ET, and we delete some custom field, we need to delete it only from event type, so event_type_id will be NULL.

Reply:
@dmitriy-uvin https://laravel.com/docs/7.x/migrations#modifying-columns

Reply:
this should work:

 $table->integer('event_type_id')->nullable()**->change()**;

Reply:
and down:

$table->integer('event_type_id')->nullable(**false**)->change();

Reply:
i did so, but i had an error with doctrine/dbal

Reply:
Just tried it locally, works correctly